scripts/versionbump.js

```diff
@@ -0,0 +1,21 @@
+const replace = require('replace-in-file');
+const pjson = require(`${process.cwd()}/package.json`);
```

Comment:
I think we don't need `pjson` here. Quick test:

`package.json`
```
 +    "process": "lerna run --stream --concurrencty 1 process"
```

`packages/node/package.json`
```
 +    "process": "node -e \"console.log(process.cwd())\""
```

```
raven-js:(master) λ: lerna run process
yarn run v1.7.0
$ node -e "console.log(process.cwd())"
/Users/kamilogorek/Projects/sentry/raven-js/packages/node
lerna success run Ran npm script 'process' in packages:
```